raiden/transfer/mediated_transfer/events.py

```diff
@@ -44,7 +45,6 @@ class SendLockedTransfer(SendMessageEvent):
     transfer: LockedTransferUnsignedState
 
     def __post_init__(self) -> None:
-        super().__post_init__()
```

Comment:
i don't think it's a good idea to remove those. If we ever add a `__post_init__` to one of the superclasses it silently won't be called. 

Instead I'd suggest to just keep an empty one in `Event`.